packages/scratch-gui/test/unit/util/project-fetcher-hoc.test.jsx

```diff
@@ -60,9 +60,10 @@ describe('ProjectFetcherHOC', () => {
         );
         storage.load = originalLoad;
         // nextTick needed since storage.load is async, and onFetchedProject is called in its then()
-        process.nextTick(
+        setTimeout(
```

Comment:
For some reason, `process.nextTick` wasn't delaying long enough. Waiting for 1ms seems to work. It wouldn't surprise me if this turns out to be flaky, though. These tests should ideally be event-driven (watch for a "load finished" event or something), but that'd be a larger change than I wanted to integrate into this PR.